main.go

```diff
@@ -33,23 +31,13 @@ func startApi() {
 }
 
 func startBlockUnlocker() {
-	u := payouts.NewBlockUnlocker(&cfg.BlockUnlocker, backend, &cfg.Network)
-	u.Start()
+	// u := payouts.NewBlockUnlocker(&cfg.BlockUnlocker, backend, &cfg.Network)
+	// u.Start()
 }
 
 func startPayoutsProcessor() {
-	u := payouts.NewPayoutsProcessor(&cfg.Payouts, backend)
-	u.Start()
-}
-
-func startNewrelic() {
-	if cfg.NewrelicEnabled {
-		nr := gorelic.NewAgent()
-		nr.Verbose = cfg.NewrelicVerbose
-		nr.NewrelicLicense = cfg.NewrelicKey
-		nr.NewrelicName = cfg.NewrelicName
-		nr.Run()
-	}
+	// u := payouts.NewPayoutsProcessor(&cfg.Payouts, backend)
```

Comment:
Please remove the references to payouts in a separate commit so that we can track it instead of commenting the code

Reply:
https://github.com/dominant-strategies/go-quai-stratum/pull/8#discussion_r1115931194

Reply:
https://github.com/dominant-strategies/go-quai-stratum/issues/10